### PR TITLE
update pytest-cov and its dependencies

### PR DESCRIPTION
**Pull request by naveen521kk**
- mingw-w64-python-attrs
- mingw-w64-python-iniconfig
- mingw-w64-python-more-itertools
- mingw-w64-python-py
- mingw-w64-python-pytest-cov

### CODE REVIEW

File: mingw-w64-python-py/PKGBUILD [hunk 1]
```diff
@@ -12,15 +12,13 @@ pkgdesc='library with cross-python path, ini-parsing, io, code, log facilities (
 arch=('any')
 url='https://pylib.readthedocs.io/'
 license=('MIT')
-depends=("${MINGW_PACKAGE_PREFIX}-python"
-         "${MINGW_PACKAGE_PREFIX}-python-iniconfig"
```
Comment (lazka) [inline mingw-w64-python-py/PKGBUILD:16]: this package seems to remove the vendoring, so this looks wrong

Comment (naveen521kk) [reply mingw-w64-python-py/PKGBUILD:16]: well, I didn't notice this. I will change this, to how it was previously. Or I will remove vendored dependencies.

Comment (lazka) [reply mingw-w64-python-py/PKGBUILD:16]: as you wish

Comment (naveen521kk) [reply mingw-w64-python-py/PKGBUILD]: done. see https://github.com/naveen521kk/msys2-updator/commit/e4737a7bc21e5fb797cb5f6805a695347857ee83

**Comment by naveen521kk** [comment]
well somethings went worse and got force pushed the same branch and closed this PR. Will create a new one...

**Pull request closed without merging**